lib/SPIRV/libSPIRV/SPIRVOpCode.h

```diff
@@ -74,6 +74,10 @@ inline bool isLogicalOpCode(Op OpCode) {
   return (unsigned)OpCode >= OpLogicalEqual && (unsigned)OpCode <= OpLogicalNot;
 }
 
+inline bool isRelationalOpCode(Op OpCode) {
```

Comment:
What do you think about `isUnaryPredicateOpCode` name for the function?

Reply:
Hmm, okay.

Reply:
Done.